packs/iconics/nahoa-level-1.json

```diff
@@ -0,0 +1,3350 @@
s pain filled him, Nahoa despaired—was this all he really could do? He hadn’t expected to defeat Pāmalō, or even survive, but he had hoped he might at least wound the creature, to buy a few years’ peace for his island. To show his people that this ancient horror could bleed, that it could be opposed, that it didn’t have to be accepted as the price for the turning of the years. But it was for naught. Demonic shadows flooded into Nahoa’s chest like a venom, and as Nahoa looked up at the sky above him, red filled his vision.</p><p>No, it didn’t. The sky above him was indeed red as blood, but it was not Nahoa’s. In the distance, Nahoa saw an impossible vision of two figures—a beast and a warrior as well—standing miles tall, bodies stretching from below the horizon to above the sky. The gods, Nahoa thought in shock, moments before one bisected the other, cutting immortality short. The heavens wept as the divine blood of the warrior god rained through the sky in a shower of sparks.</p><p>The sparks screamed past Nahoa and Pāmalō in streaks of red and silver. One landed on Pāmalō’s wing, and in a rush of speed, the sea below was replaced by the red desert of the distant mainland. Another landed, and the desert was replaced by ocean anew, though a strange one, studded through with the remnants of shattered land and ruined towers.</p><p>A spark screamed through the air, shining with light, and its speed matched Pāmalō and Nahoa’s as they flew. The demon tore into Nahoa once more, and Nahoa clung desperately to consciousness as bone crunched. He looked to the spark. He had been Called to protect his island, to face this beast, to do something, anything, but intention was nothing without the power to see it through. Power that hung in the air now, right before him. Nahoa reached for the spark with his right hand, but Pāmalō held him in place. He reached again, and his flesh tore to beak and talon. With a cry of pain, Nahoa reached a third time, and he seized divinity.</p><p>The spark flooded deep into Nahoa’s soul. He felt death flee from his bones as divine power chased the shadows from his veins, burning them into his skin as tattoos. His power, now, as well. Nahoa wrestled Pāmalō’s talons open, finding them brittle in his grasp, and he reached out for a weapon, finding the spear dropped half a world ago in his grip once more. As he yelled a challenge anew to the creature that had terrorized his people, he felt the tiny spark of divine power within him blaze, an ember fanned into a bonfire by this single moment of glory. Nahoa thrust the spear deep into Pāmalō’s breast, and the might of gods coursed down his arm and through the weapon to detonate in a peal of light and thunder. As the demon’s torn body fell to earth, Nahoa fell with it, careening toward a strange, icy shore.</p><p>Nahoa was no hero, not yet. But as consciousness slipped from his grasp, the Calling rang out around him, clearer than ever, and he knew it was what he was destined to be.</p>",
```

Comment:
```suggestion
                "backstory": "<p>"Nahoa, can you help me dig another ditch before planting season?"</p><p>"Nahoa, the pigs have gotten out again—can you help me round them up?"</p><p>The Olehala people, third of the four great nations of the Okaiyo Ocean, had many heroes. Kolea the Wanderer, or Uncle Midnight, or the Tideweaver. But Nahoa's village, on a small island in the easternmost reaches of Olehala's archipelago, had none. It didn't need any; the village had plentiful rain and bountiful harvests, the seas were calm and the currents easy. But for whatever small problems might crop up—a ripped sail here, a sick child in need of watching there—Nahoa was always first to pitch in. The young man was no hero, but he worked hard on his family's taro farm, sharing whatever was left over with his neighbors, and he played harder, stone-bowling with friends or lava-sledding down the slopes.</p><p>The years were joyful. Planting and harvesting, watching his little sisters grow up. The New Year's games and feasts. Lazy summers swimming in the falls with that navigator from the nearby Taotake islands, before she led her merchant canoes east to distant Arcadia. For many years, all was well.</p><p>Until Pāmalō returned. The Olehala's chants told of the demon, a massive creature of shadow and smoke, its wings silent as an owl and its belly insatiable as a shark's. Once a generation, it would appear in the skies above their island, roost on the highest mountain for one day and night, and then feed. As the creature alit and the skies darkened, the people did what they had always done and hid: some in caves, some in the forests, and some in the distant reefs. With luck, Pāmalō would take mostly livestock, and leave.</p><p>Nahoa couldn't understand. How could they let this happen, generation after generation? The last time Pāmalō had come, Nahoa had been too little to remember much, but the fear of black wings against the sun had been seared in his mind, the same as everyone of the nameless, easternmost island. Someone had to do something. His heart was telling Nahoa to do something.</p><p>No… it was Calling.</p><p>The Call had rung out for decades, and whether the others could not hear it or could not answer, Nahoa did not know. Without words, it implored someone to help. Nahoa was no hero… but he always helped those in need, and so he did what he always did when asked to help.</p><p>That night, Nahoa bathed in the waterfall and tidied the garden. He prepared some of his sisters' favorite food, leaving it near the cooking pit for them to find in the morning, and he picked flowers to circle the stone that remembered their mother and father. Lastly, he went to the path that led up the mountain, digging stick in hand. Nahoa sat on a stump, the farming tool over his knee. So my sisters can grow up without fear. He cut a notch into his digging stick. So Uncle doesn't go hungry this winter. Another notch. So that nice Taotake girl doesn't encounter it on the seas. With each reason to face the creature, he cut. When he'd finished counting all his reasons, his digging stick was a barbed spear. With it, he climbed the mountain to Pāmalō's roost, and he cried out a challenge as dawn broke.</p><p>Pāmalō paid no regard. Of course it didn't. It was ancient, the subject of songs and dances every young Olehalan learned from their grandparents, its face carved into warding stones generations ago. Nahoa was no hero. The creature snatched Nahoa in its talons, and the two left the earth behind.</p><p>High in the sky, Nahoa fought Pāmalō. He may as well have been an ant attempting to fight the tide, for all the difference it seemed to make. Nahoa thrust his spear forward and it skittered along the demon owl's slick, black feathers, to fall to the raging waves below; he seized its wing to force it to earth but found it unyielding as a koa tree. He gave a great shout and it looked back at him, at this insignificant thing that dared to fight back, and it cocked its head. Nahoa didn't even have time to let out a gasp as his chest tore under its beak.</p><p>As pain filled him, Nahoa despaired—was this all he really could do? He hadn't expected to defeat Pāmalō, or even survive, but he had hoped he might at least wound the creature, to buy a few years' peace for his island. To show his people that this ancient horror could bleed, that it could be opposed, that it didn't have to be accepted as the price for the turning of the years. But it was for naught. Demonic shadows flooded into Nahoa's chest like a venom, and as Nahoa looked up at the sky above him, red filled his vision.</p><p>No, it didn't. The sky above him was indeed red as blood, but it was not Nahoa's. In the distance, Nahoa saw an impossible vision of two figures—a beast and a warrior as well—standing miles tall, bodies stretching from below the horizon to above the sky. The gods, Nahoa thought in shock, moments before one bisected the other, cutting immortality short. The heavens wept as the divine blood of the warrior god rained through the sky in a shower of sparks.</p><p>The sparks screamed past Nahoa and Pāmalō in streaks of red and silver. One landed on Pāmalō's wing, and in a rush of speed, the sea below was replaced by the red desert of the distant mainland. Another landed, and the desert was replaced by ocean anew, though a strange one, studded through with the remnants of shattered land and ruined towers.</p><p>A spark screamed through the air, shining with light, and its speed matched Pāmalō and Nahoa's as they flew. The demon tore into Nahoa once more, and Nahoa clung desperately to consciousness as bone crunched. He looked to the spark. He had been Called to protect his island, to face this beast, to do something, anything, but intention was nothing without the power to see it through. Power that hung in the air now, right before him. Nahoa reached for the spark with his right hand, but Pāmalō held him in place. He reached again, and his flesh tore to beak and talon. With a cry of pain, Nahoa reached a third time, and he seized divinity.</p><p>The spark flooded deep into Nahoa's soul. He felt death flee from his bones as divine power chased the shadows from his veins, burning them into his skin as tattoos. His power, now, as well. Nahoa wrestled Pāmalō's talons open, finding them brittle in his grasp, and he reached out for a weapon, finding the spear dropped half a world ago in his grip once more. As he yelled a challenge anew to the creature that had terrorized his people, he felt the tiny spark of divine power within him blaze, an ember fanned into a bonfire by this single moment of glory. Nahoa thrust the spear deep into Pāmalō's breast, and the might of gods coursed down his arm and through the weapon to detonate in a peal of light and thunder. As the demon's torn body fell to earth, Nahoa fell with it, careening toward a strange, icy shore.</p><p>Nahoa was no hero, not yet. But as consciousness slipped from his grasp, the Calling rang out around him, clearer than ever, and he knew it was what he was destined to be.</p>",
```
Careful of the non-standard characters like rsquo